src/components/sequencing/actions/ActionRunCard.svelte

```diff
@@ -0,0 +1,84 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import PlayBtnIcon from 'bootstrap-icons/icons/play-btn.svg?component';
+  import StopwatchIcon from 'bootstrap-icons/icons/stopwatch.svg?component';
+  import { Status } from '../../../enums/status';
+  import type { ActionDefinition, ActionRun, ActionRunSlim } from '../../../types/actions';
+  import { formatMS } from '../../../utilities/time';
+  import StatusBadge from '../../ui/StatusBadge.svelte';
+
+  export let actionRun: ActionRunSlim;
+  export let actionDefinition: ActionDefinition | null;
+  export let interactable: boolean = true;
+
+  $: console.log('actionRun :>> ', actionRun);
+
+  function getStatusForActionRun(actionStatus: ActionRun['status']): Status {
+    if (actionRun.error || actionRun.results?.status === 'FAILED') {
```

Comment:
The type definition for `status` has lower case "failed". Is there a chance that either lowercase or uppercase will be returned?

Reply:
Actually n/m. The `results` is typed `any`. Is it possible to type it further?